components/dashboard/components/TimelineChart.jsx

```diff
@@ -24,7 +24,8 @@ const drawGantt = (props) => {
     // create map of id: name
     // local user will always be first.
     logger.debug('sorted participants:', participants);
-    const participantNames = participants.map((p) => p.name);
+    const truncateLength = props.width < 720 ? 13 : 18;
```

Comment:
You've introduced 3 magic numbers; 720, 13 and 18. What are they and where do those particular values come from?
Define them as const values and use the names, and possibly add some additional comment where the const is defined if the name isn't informative enough by itself.

Also I saw that you changed the 2nd 2 values from 15 and 20 when you modified the textTruncate function to default to a single ellipsis character instead of 3 periods, but I don't understand why that should change these values.

Reply:
I guess, as you mentioned earlier, different characters can have different pixel widths. It seemed that the ellipsis character took up almost as much horizontal space as three periods. 

I have updated to using informative constants for these values.